### ISSUE TITLE
DF Chat Enhancements: Public Roll icon is identical to core icon

### ISSUE THREAD

**Issue by weepingminotaur** [flamewave000/dragonflagon-fvtt#415]
When using the "Roll Buttons: Replace Roll Type Selector with Buttons" option, the Public Roll icon (dice) is identical to the core Foundry icon (dice) that appears just to its left. This is confusing for users. A different Public Roll icon would solve the problem.

**Comment by darloth** [comment]
Does that icon serve a purpose?  Is it appropriate for a module to flat out remove it?  If so, perhaps that would be another potential solution?

**Comment by weepingminotaur** [comment]
Yeah, it doesn't seem to actually do anything, so maybe replacing it would be good (and free up space visually, never a bad thing).

**Comment by flamewave000** [comment]
@darloth @weepingminotaur I originally wanted to remove it, but Dice Tray actually uses it to display their Dice Calculator tool, and given that module's popularity I didn't want to create a conflict. I didn't really think of it as being all that confusing though since one is an icon and the other is an actual button you can press.

**Comment by oakbrad** [comment]
This is indeed very confusing for some of my players since the icon is the same. We are using Dice Tray, so clicking the Foundry D20 is not just an icon - it's the button that opens the dice notation calculator. So there's two D20 buttons right next to each other that do different things. 

![Screen Shot 2023-03-07 at 13 35 55](https://user-images.githubusercontent.com/24882215/223558574-0b5c3998-7720-41bc-ad9f-417dff4b6b7a.png)

I think using the D20 icon is not clear to the purpose of the 'Public Roll' setting. The D20 is used all over the Foundry UI to indicate an actual dice roll command (so makes sense for the Dice Tray calc to use it). The current GM, Blind, and Self icons all visually communicate the level of visibility, I would propose changing the 'Public Roll' icon to match that intention. I think [MDI account-group](https://pictogrammers.com/library/mdi/icon/account-group/) would be a better choice, indicating that the roll is visible to all vs GM vs secret.

![account-group](https://user-images.githubusercontent.com/24882215/223559436-60e9e283-d1a4-47cf-9d32-7e27f54b6cec.png)

**Comment by flamewave000** [comment]
Since Foundry only supports Font Awesome icons, I can convert it to this icon: "fas fa-users" 

https://fontawesome.com/search?q=users&o=r
